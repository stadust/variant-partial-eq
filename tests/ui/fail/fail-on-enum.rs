use variant_partial_eq::VariantPartialEq;

#[derive(VariantPartialEq)]
enum MyEnum {}

fn main() {}