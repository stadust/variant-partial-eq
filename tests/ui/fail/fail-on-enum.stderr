error: #[derive(VariantPartialEq)] only works for structs
 --> tests/ui/fail/fail-on-enum.rs:3:10
  |
3 | #[derive(VariantPartialEq)]
  |          ^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `VariantPartialEq` (in Nightly builds, run with -Z macro-backtrace for more info)
